mmv1/third_party/terraform/resources/resource_compute_instance.go.erb

```diff
@@ -1004,6 +1034,11 @@ func expandComputeInstance(project string, d *schema.ResourceData, config *Confi
 		return nil, fmt.Errorf("Error creating scheduling: %s", err)
 	}
 
+        params, err := expandParams(d)
```

Comment:
nit: Do you mind renaming associated expand / flatten funcs to namespace on the resource, similar to https://github.com/hashicorp/terraform-provider-google/blob/53295dc35039ca8cd66da18cc758bf7487bd8c03/google/resource_compute_address.go#L195-L200?

This file predates that convention, unfortunately, so there are counterexamples present.

Reply:
It makes sense to me but in this folder I guess it would be better to follow the same name convention as we can see "expandBootDisk" etc. here?

Reply:
Ah- I guess the whole file's consistent in not doing so. SGTM.